osgar/explore.py

```diff
@@ -0,0 +1,146 @@
+"""
+  Explore - follow a wall/obstacles
+    (use 2D SICK LIDAR only)
+"""
+import math
+from datetime import timedelta
+
+import numpy as np
+
+from osgar.node import Node
+
+
+SAFE_DISTANCE_STOP = 0.5
+SAFE_DISTANCE_GO = SAFE_DISTANCE_STOP + 0.3
+WALL_DISTANCE = 0.7  # m
+DESIRED_SPEED = 0.4  # m/s
+
+
+def min_dist(data):
+    data = np.array(data)
+    mask = (data > 100)
```

Comment:
What is expected to be inside `data`? Why is it a good idea to consider only items > 100?